package.json

```diff
@@ -28,6 +28,7 @@
     "pretty-bytes": "^3.0.1",
     "gm": "^1.23.0",
     "json-autosave": "~1.1.2",
+    "jsonpath": "^0.2.12",
```

Comment:
In general, prefer `~` over `^` as it's safer. However it looks like there's a mixture of both in here, so this is fine.

Reply:
I'll update it to be `~` better to have it right before its merged